dbt_meshify/dbt_projects.py

```diff
@@ -432,6 +459,39 @@ def _get_custom_macros(self) -> Set[str]:
                 macros_set.update(self._get_macro_dependencies(macro))
         return macros_set
 
+    @property
+    def referenced_docs(self) -> Set[str]:
+        """Return a list of all docs referenced within this SubProject."""
+
+        if self._referenced_docs:
+            return self._referenced_docs
+
+        docs = set()
+        for unique_id in self.resources:
+            if unique_id.startswith("test."):
+                continue
+
+            node = self.get_manifest_node(unique_id)
+
+            if node is None:
+                continue
+
+            if hasattr(node, "raw_code"):
```

Comment:
forgetting which nodes have `raw_code` -- does this indicate we'd be searching the content of a model for docs blocks? are we looking for docs in the definition of YML only resources in this step?

Reply:
I think you've figured this out already, but yes. This checks `raw_code`, and the next check on line 482 checks wihtin the YAML file, too.